src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/ReadWriteWalker.vb

```diff
@@ -46,6 +52,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private ReadOnly _readOutside As HashSet(Of Symbol) = New HashSet(Of Symbol)()
         Private ReadOnly _writtenOutside As HashSet(Of Symbol) = New HashSet(Of Symbol)()
         Private ReadOnly _captured As HashSet(Of Symbol) = New HashSet(Of Symbol)()
+        Private ReadOnly _capturedInside As HashSet(Of Symbol) = New HashSet(Of Symbol)()
+        Private ReadOnly _capturedOutside As HashSet(Of Symbol) = New HashSet(Of Symbol)()
```

Comment:
would it worth to use `PooledHashSet` here like C#? (also I noticed we don't free them in C#)

Reply:
If we're not freeing them, there is no reason to use a pool.